src/whitenoise/compress.py

```diff
@@ -38,6 +38,19 @@ class Compressor:
         # Fonts
         "woff",
         "woff2",
+        # Video
+        "3gp",
+        "3gpp",
+        "asf",
+        "asx",
```

Comment:
This is a text playlist format so likely to be small and compressible

https://fileinfo.com/extension/asx

```suggestion
```